packages/xo-server-backup-reports/src/index.js

```diff
@@ -100,40 +102,113 @@ const isSkippedError = error =>
   error.message === UNHEALTHY_VDI_CHAIN_ERROR ||
   error.message === NO_SUCH_OBJECT_ERROR
 
-const INDENT = '  '
-const createGetTemporalDataMarkdown = formatDate => (
-  start,
-  end,
-  nbIndent = 0
-) => {
-  const indent = INDENT.repeat(nbIndent)
-
-  const markdown = [`${indent}- **Start time**: ${formatDate(start)}`]
+// ===================================================================
+
+const TITLE_BY_STATUS = {
+  failure: n => `## ${n} Failure${n === 1 ? '' : 's'}`,
+  interrupted: n => `## ${n} Interruption${n === 1 ? '' : 's'}`,
+  skipped: n => `## ${n} Skip${n === 1 ? '' : 's'}`,
+  success: n => `## ${n} Success${n === 1 ? '' : 'es'}`,
```

Comment:
I'm sorry but I changed my mind regarding this, let's keep it as it previously, we'll take a decision later.

Sorry for all these changes.